ngx_pq_module.c

```diff
@@ -162,6 +163,7 @@ typedef struct {
     ngx_pq_save_t *save;
     ngx_queue_t queue;
     ngx_uint_t type;
+    ngx_uint_t row_count;
```

Comment:
```suggestion
    ngx_uint_t rows;
```
and all members should be sorted alphanumerically

Reply:
rows - is a set of rows. rows_count is more mnemonic.

Reply:
"all members should be sorted alphanumerically" - is it "cosmetic" only? There is no alphanumeric in any structure. Should I resort them all?